src/views/v2/rsvp/ari/form.php

```diff
@@ -14,11 +14,14 @@
  * @var WP_Post|int $post_id The post object or ID.
  *
  * @since 4.12.3
+ * @since TBD Ensured that $going is not undefined in Ajax call.
  *
- * @version 5.0.0
+ * @version TBD
  */
 
+$going = $this->get( 'going' );
 ?>
+
```

Comment:
This extra line may be the reason why snapshots are failing. I'd remove it.

Reply:
@juanfra Ok.